Subject: [PATCH] refpolicy: fix real path for portmap.

Fix file contexts for portmap files to match the oe-core install
paths.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
---
 policy/modules/contrib/portmap.fc |    7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/policy/modules/contrib/portmap.fc b/policy/modules/contrib/portmap.fc
index 3cdcd9f..3faf697 100644
--- a/policy/modules/contrib/portmap.fc
+++ b/policy/modules/contrib/portmap.fc
@@ -5,12 +5,9 @@ ifdef(`distro_debian',`
 /sbin/pmap_dump		--	gen_context(system_u:object_r:portmap_helper_exec_t,s0)
 /sbin/pmap_set		--	gen_context(system_u:object_r:portmap_helper_exec_t,s0)
 ', `
-/usr/sbin/pmap_dump	--	gen_context(system_u:object_r:portmap_helper_exec_t,s0)
-/usr/sbin/pmap_set	--	gen_context(system_u:object_r:portmap_helper_exec_t,s0)
+/sbin/pmap_dump	--	gen_context(system_u:object_r:portmap_helper_exec_t,s0)
+/sbin/pmap_set	--	gen_context(system_u:object_r:portmap_helper_exec_t,s0)
 ')
 
 /var/run/portmap\.upgrade-state -- gen_context(system_u:object_r:portmap_var_run_t,s0)
-
-ifdef(`distro_debian',`
 /var/run/portmap_mapping --	gen_context(system_u:object_r:portmap_var_run_t,s0)
-')
-- 
1.7.9.5

